api/src/snekbox.py

```diff
@@ -0,0 +1,6 @@
+import requests
+
+
+def evaluate(code: str) -> str:
+    response = requests.post("http://localhost:8060/eval", json={"input": code}, timeout=2)
```

Comment:
URL should be given via env variable or access it via the docker network, eg `http://snekbox:8060` as user may choose to different ports

Reply:
Good point, I'll change that